core/consensus/grandpa/gossiper.hpp

```diff
@@ -20,11 +20,9 @@ namespace kagome::consensus::grandpa {
   struct Gossiper {
     virtual ~Gossiper() = default;
 
-    virtual void precommit(Precommit pc) = 0;
+    virtual void vote(const VoteMessage &msg) = 0;
 
-    virtual void prevote(Prevote pv) = 0;
-
-    virtual void primaryPropose(PrimaryPropose pv) = 0;
+    virtual void fin(const Fin &fin) = 0;
```

Comment:
Also, I understand that it's the name from the spec so it makes total sense to keep it this way, but, if I understand correctly, this is a shortcut for 'finalize', and (especially as 'fin' itself is an English word for a part of a fish) I think there's no need for such shortcuts in the code. In the spec they are useful in math-like pseudocode, but in actual code any shortage just makes it a bit harder to read.